python/cudf/cudf/options.py

```diff
@@ -112,3 +114,52 @@ def describe_option(name: Optional[str] = None):
     names = _OPTIONS.keys() if name is None else [name]
     for name in names:
         print(_build_option_description(name, _OPTIONS[name]))
+
+
+def _make_contains_validator(valid_options: Container) -> Callable:
+    """Return a validator that checks if a value is in `valid_options`."""
+
+    def _validator(val):
+        if val not in valid_options:
+            raise ValueError(
+                f"{val} is not a valid option. "
+                f"Must be one of {set(valid_options)}."
+            )
+
+    return _validator
+
+
+_register_option(
+    "default_integer_bitwidth",
```

Comment:
In hindsight, `default_integer_size` sounds more familiar than `default_integer_bitwidth`.

Reply:
We will keep it as `bitwidth` for now and open to future naming requests if asked.